docs/administration/upgrade/upgrade-test.md

```diff
@@ -7,15 +7,15 @@ weight: 50
 1. Add user usera userb userc userd usere, set usera userb as system admin.  
 2. Create project projecta projectc as private, create projectb as public.  
 3. Add usera as projecta's admin, userc as developer, and userd as guest. Do the same to projectb and projectc.  
-4. Login harbor as usera, push an unsigned image into projecta, then push a signed image to projecta. 
+4. Login harbor as usera, push an unsigned image into projecta, then push a signed image to projecta.
```

Comment:
user a

Reply:
usera is the name of the users from the instructions above.